src/actions/index.js

```diff
@@ -1,60 +1,61 @@
 export function loadMyMovieList(dispatch) {
-    dispatch({
-        type:"LOAD_MY_MOVIE_LIST"
-    });
-
-    fetch("/movies")
-    .then( (response) => {
-        return response.json();
-    }).then((movies) => {
-        dispatch(myMovieListLoaded(movies));
-    });
+  dispatch({
```

Comment:
need to double check this. Action functions always need to return something. Look at what loadSearch is returning